ci_framework/roles/devscripts/tasks/sub_tasks/_firewall.yml

```diff
@@ -0,0 +1,68 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Configure firewall settings for all extra networks
+  become: true
+  ansible.posix.firewalld:
+    zone: libvirt
+    interface: "{{ item }}"
+    state: enabled
+    permanent: true
+    immediate: true
+  loop: "{{ cifmw_devscripts_extra_networks }}"
+
+# cifmw_devscripts_osp_compute_nodes would be defined only if there
+# are physical servers acting as computes. In case of virtual, this
+# variable is only populated by 43_add_bmh.yml
+- name: Masquerade when virtual network extends to physical space
+  when:
+    - cifmw_devscripts_osp_compute_nodes is defined
+    - cifmw_devscripts_osp_compute_nodes | length > 0
+  block:
+    - name: Regather network information
+      ansible.builtin.setup:
+        gather_subset:
+          - default_ipv4
+          - default_ipv6
+
+    - name: Ensure masquerading for external network
+      become: true
+      vars:
+        ext_interface: >-
+          {{
+            ansible_default_ipv4.interface
+            if cifmw_devscripts_config.ip_stack == 'v4'
+            else
+            ansible_default_ipv6.interface
+          }}
+      ansible.builtin.iptables:
```

Comment:
if you don't set the `ip_version` it will defaults to `ipv4` - meaning it won't be applied as you may think:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/iptables_module.html#parameter-ip_version

Reply:
Thanks Cedric, since we are moving towards `nft` and the current supported is ipv4. Would it be okay if I can pick it up later?